Demo/UIKit/Info.plist

```diff
@@ -79,6 +79,8 @@
 		<key>NSAllowsLocalNetworking</key>
 		<true/>
 	</dict>
+	<key>NSPhotoLibraryAddUsageDescription</key>
+	<string>Save vouchers to your photos library</string>
```

Comment:
We better not forget to mention this in our docs :D

Reply:
Yeah, true, I'll add it to the readme file.

Reply:
added a paragraph to the ReadME file.